src/js/scorecard.js

```diff
@@ -0,0 +1,64 @@
+function Scorecard() {
+  this.points = [];
+  this.pointsIndex = 0;
+  this.STRIKE = 10;
+  this.SPARE = 10;
+}
+
+Scorecard.prototype.roll = function(rollPoints) {
+  this.points.push(rollPoints);
+};
+
+Scorecard.prototype.framescore = function(points) {
```

Comment:
Good separation of concerns by using separate classes for `isStrike()`, `isSpare()`, `strikeScore()`, `spareScore()` etc.